### ISSUE TITLE
Efficiency of last stage

### ISSUE THREAD

**Issue by triangleCZH** [THUKey/pytorch-mobilenet-v3#7]
I implemented v3 and tested with/without the modified last stage settings. Although paper claims that this modification doesn't affect accuracy, I find out accuracy actually drops (i.e. 73.45->71.84). May I know about your opinion on that? Thank you!

**Comment by triangleCZH** [comment]
Ok. When training for enough epochs, efficient last stage doesn't harm the accuracy.
